resources/locale/en.yml

```diff
@@ -1,4 +1,35 @@
 fof-nightmode:
+    common:
```

Comment:
This should be `ref`, not `common` - see https://flarum.org/docs/extend/i18n.html#appendix-a-standard-key-format.